XCUITests/ThirdPartySearchTest.swift

```diff
@@ -63,93 +56,40 @@ class ThirdPartySearchTest: BaseTestCase {
     }
 
     func testCustomSearchEngineDeletion() {
-        // Visit MDN to add a custom search engine
-        loadWebPage(mozDeveloperWebsite, waitForLoadToFinish: true)
-        addSearchProvider()
-
-        if !isTablet {
-            dismissKeyboardAssistant(forApp: app)
-        }
-
-        let tabTrayButton = self.tabTrayButton(forApp: app)
-        tabTrayButton.tap()
-        app.buttons["TabTrayController.addTabButton"].tap()
-        waitForExistence(app.textFields["url"], timeout: 3)
+        navigator.performAction(Action.AddCustomSearchEngine)
+        app.navigationBars["Search"].buttons["Settings"].tap()
+        app.navigationBars["Settings"].buttons["AppSettingsTableViewController.navigationItem.leftBarButtonItem"].tap()
         app.textFields["url"].tap()
         waitForExistence(app.buttons["urlBar-cancel"])
         app.typeText("window")
```

Comment:
As above, but in this case we don't have to go to the url, so only Paste is needed:
```
UIPasteboard.general.string = "window"
app.textFields.firstMatch.press(forDuration: 1)
app.staticTexts["Paste"].tap()
```